src/js/components/TaskDetail.js

```diff
@@ -1,21 +1,26 @@
-import classNames from 'classnames';
+import mixin from 'reactjs-mixin';
 /* eslint-disable no-unused-vars */
 import React from 'react';
 /* eslint-enable no-unused-vars */
+import {StoreMixin} from 'mesosphere-shared-reactjs';
 
 import DescriptionList from './DescriptionList';
+import MarathonStore from '../stores/MarathonStore';
 import MesosStateStore from '../stores/MesosStateStore';
 import MesosSummaryStore from '../stores/MesosSummaryStore';
 import ResourceTypes from '../constants/ResourceTypes';
 import RequestErrorMsg from './RequestErrorMsg';
-import SidePanelContents from './SidePanelContents';
+import ServicesBreadcrumb from './ServicesBreadcrumb';
+import PageHeader from './PageHeader';
 import TaskDebugView from './TaskDebugView';
 import TaskDirectoryView from './TaskDirectoryView';
 import TaskDirectoryStore from '../stores/TaskDirectoryStore';
 import TaskStates from '../constants/TaskStates';
-import TaskUtil from '../utils/TaskUtil';
 import Units from '../utils/Units';
 
+import InternalStorageMixin from '../mixins/InternalStorageMixin';
+import TabsMixin from '../mixins/TabsMixin';
+
 const TABS = {
   files: 'Files',
   details: 'Details',
```

Comment:
Lets move `details` so it's the first one, as we now assume that to be the case. Then we don't need to explicitly assing the string value to the `this.state.currentTab` at line `37`.